pkg/metrics/collector/kernel_metrics.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	tcpTimeoutDesc = prometheus.NewDesc(
+		"tcp_timeout_rehash_count",
+		"Tcp timeout rehash, collected from nstat",
```

Comment:
No need to mention the source of this data here. (Also, it's already outdated...)

Reply:
Done